src/components/Favicon/Favicon.stories.mdx

```diff
@@ -26,16 +26,16 @@ export const Template = (args) => <Favicon {...args} />;
 <Canvas>
   <Story name="Default">{Template.bind({})}</Story>
   <Story name="Examples">
-    <Favicon domain={'physicsworld.com'} />
-    <Favicon domain={'bbc.co.uk'} />
-    <Favicon domain={'politifact.com'} />
-    <Favicon domain={'fullfact.org'} />
-    <Favicon domain={'usatoday.com'} />
-    <Favicon domain={'factchecker.in'} />
-    <Favicon domain={'verafiles.org'} />
-    <Favicon domain={'newsmobile.in'} />
-    <Favicon domain={'faktisk.no'} />
-    <Favicon domain={'verifica.efe.com'} />
+    <Favicon rootDomain={'physicsworld.com'} />
+    <Favicon rootDomain={'bbc.co.uk'} />
+    <Favicon rootDomain={'politifact.com'} />
+    <Favicon rootDomain={'fullfact.org'} />
+    <Favicon rootDomain={'usatoday.com'} />
+    <Favicon rootDomain={'factchecker.in'} />
+    <Favicon rootDomain={'verafiles.org'} />
+    <Favicon rootDomain={'newsmobile.in'} />
```

Comment:
With this favicon I see this in storybook ![image](https://user-images.githubusercontent.com/22867872/212919871-15fd323d-a9d4-435b-8d78-dcdf30b644f6.png)

while it should be this 
![image](https://user-images.githubusercontent.com/22867872/212920245-efa82ac4-e84f-41c2-ba6f-11ae7fb84649.png)

Could you check it?

Reply:
Weird, this one and a couple more also didn't load for me, even though the url is correct, but after reloading it shows up. Maybe some caching issue.
Can you try if it shows up for you when you reload the page?

Reply:
Yeah, fixed with reload